### PR TITLE
update vi-bellapps.md, rbts-intern-orientation.md, vi-bellapps.md

### LINKED ISSUE #1690: Community updates issues
In http://open-learning-exchange.github.io/#!pages/vi/vi-bellapps.md
Section "Different Kinds of Updates to Your Community"

For me it didn't work to manually update (status => ) update publications in Chrome. 

It did first work when I did use Firefox. That could be the issue, or just logging in again perhaps?

Early on in the Steps, it is mentioned that Firefox is the way to go, but in the same section (again: "Different Kinds of Updates to Your Community") Chrome is mentioned, so I assumed it would be okay to use?

### PR DESCRIPTION
**Pull request by xyb994**
- vi-first-steps.md: use vi+resume@ole.org
- rbts-intern-orientation.md: update current projects section and calendar subscription section
- vi-bellapps.md: general clean up and reorganize

#### Rawgit link
https://rawgit.com/open-learning-exchange/open-learning-exchange.github.io/update-bell-apps-tutorial/#!pages/vi/vi-first-steps.md

https://rawgit.com/open-learning-exchange/open-learning-exchange.github.io/update-bell-apps-tutorial/#!pages/robots/rbts-intern-orientation.md

https://rawgit.com/open-learning-exchange/open-learning-exchange.github.io/update-bell-apps-tutorial/#!pages/vi/vi-bellapps.md

Close #1690

### CODE REVIEW

**Comment by kylemathias** [comment]
let us know when we should review.

File: pages/robots/rbts-intern-orientation.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ You should check if the following conditions are met:
 * [x] include issue number in pull request title
 * [x] no unneeded files/lines change in pull request
```
Comment (Jackbui96) [inline pages/robots/rbts-intern-orientation.md]: I think we should rewrite this to: "don't include unnecessary files/lines change in the pull request"

File: pages/vi/vi-bellapps.md [hunk 2]
```diff
@@ -3,65 +3,76 @@
 ## Objectives
 
 * Understand how the BeLL-Apps interface is structured
```
Comment (Jackbui96) [inline pages/vi/vi-bellapps.md]: How's about we use these phase instead? "Understand how Bell-Apps interface is structured"?

File: pages/robots/rbts-intern-orientation.md [hunk 3]
```diff
@@ -50,8 +59,8 @@ You should check if the following conditions are met:
...
 * [x] there's a branch for the patch
```
Comment (Jackbui96) [inline pages/robots/rbts-intern-orientation.md]: Can we change to this instead: "Create a new branch for the patch"?

File: pages/robots/rbts-intern-orientation.md [hunk 4]
```diff
@@ -50,8 +59,8 @@ You should check if the following conditions are met:
...
-* [x] the pull request actually fix the issue
-* [x] it display without any errors on rawgit
+* [x] the pull request actually fixes the issue
+* [x] it displays without any errors on rawgit
```
Comment (Jackbui96) [inline pages/robots/rbts-intern-orientation.md]: Can we change to this instead? "No display errors on rawgit preview"? The usage of "it displays" here is ambiguous.

File: pages/robots/rbts-intern-orientation.md [hunk 5]
```diff
@@ -50,8 +59,8 @@ You should check if the following conditions are met:
...
 * [x] no merge conflicts
 * [x] commits are associated with GitHub account
```
Comment (Jackbui96) [inline pages/robots/rbts-intern-orientation.md]: Just a grain of salt advice. Should we said something like: "commits are associated with __your__ Github account"?

**Pull request merged**